src/Agent/NewRelic/Agent/Core/OpenTelemetryBridge/ActivityBridge.cs

```diff
@@ -83,7 +83,7 @@ private void TryCreateActivityListener()
 
             _activityListener = Activator.CreateInstance(activityListenerType);
 
-            ConfigureShouldListenToCallback(_activityListener, activityListenerType, activitySourceType);
+            ConfigureShouldListenToCallback(this, _activityListener, activityListenerType, activitySourceType);
```

Comment:
This method doesn't need to be static so that it can directly access `this` within the method.